src/components/Promo/PromoCard.tsx

```diff
@@ -95,10 +98,9 @@ export const PromoCard = () => {
       },
     )
   }, [handleClick])
-
-  return (
+  return renderPromos.length > 0 ? (
```

Comment:
If you feel like clicking this button:

```suggestion
  return renderPromos.length ? (
```